.changeset/dull-candles-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': minor
+---
+
+Adds an adapter option to configure the used image service. Read more in our Cloudflare adapter docs: https://docs.astro.build/en/guides/integrations-guide/cloudflare/
```

Comment:
```suggestion
Adds an `imageService` adapter option to configure which image service is used. Read more in the [Cloudflare adapter docs](https://docs.astro.build/en/guides/integrations-guide/cloudflare/).
```
When you add an option, it's often nice to specify what it is so people can easily find it!